test/functional/services/browser.js

```diff
@@ -65,6 +65,7 @@ export async function BrowserProvider({ getService }) {
       const current = await driver.getCurrentUrl();
       const currentWithoutTime = modifyUrl(current, parsed => {
         delete parsed.query._t;
+        return undefined;
```

Comment:
I can see similar changes in #33494 
Let's leave it there for `browser.js`, `remote.ts` and `webdriver.ts` unless it is required by newly added  tests